examples/vanilla-ts-esm/public/mux-player.html

```diff
@@ -59,11 +60,12 @@
       metadata-video-id="video-id-example"
       metadata-video-title="Video Title Example"
       metadata-viewer-user-id="user-id-example"
+      secondary-color="rgba(0, 255, 100, 0.3)"
      -->
 
     <mux-player
       stream-type="on-demand"
-      playback-id="g65IqSFtWdpGR100c2W8VUHrfIVWTNRen"
+      playback-id="026vljfeZ99KwUPRbl5YHVdCTSrBsPfxjkyM3PSlfWGw"
```

Comment:
nit (non-blocking): did you mean to keep this changed playback-id?

Reply:
yes, the letterboxing of the Mad Max clip is a bit annoying working on the controls UI